packages/mux-player/src/template.ts

```diff
@@ -114,8 +109,31 @@ export const ChromeRenderer = (props: MuxTemplateProps) => {
 };
 
 // prettier-ignore
-const MediaPlayButton = () => html`
-  <media-play-button>
+const SeekToLiveButton = (props: Partial<MuxTemplateProps>) => html`
+  <button 
+    aria-disabled="${props.inLiveWindow}"
+    in-live-window="${props.inLiveWindow}"
+    onclick="${function (this: HTMLButtonElement, evt: Event) {
+      props.onSeekToLive?.(evt);
+      if (props.paused) {
+        const playRequestEvt = new CustomEvent(
+          'mediaplayrequest', 
+          { composed: true, bubbles: true }
+        );
+        (this as HTMLButtonElement).dispatchEvent(playRequestEvt);
+      }
+    }}" 
+    class="mxp-seek-to-live-button"
+  >
+    Live
+  </button>
+`;
+
+// prettier-ignore
+const MediaPlayButton = (props: Partial<MuxTemplateProps>) => html`
+  <media-play-button 
+    onmediaplayrequest="${props.streamType === StreamTypes.LIVE || props.streamType === StreamTypes.LL_LIVE ? props.onSeekToLive : false}"
```

Comment:
blocking: I think this line can be removed, the prop onSeekToLive can be removed, in the above onclick handler too.

the seekToLive function can be called in the `play` event listener of the media element if needed.

Reply:
cool this has product implications. Chatting w/ @dylanjha about them now. As long as we're ok with the broader assumptions here, I'll refactor.

the seek to live button does not _*necessarily*_ also play. E.g., if content is currently playing but is not in the live window, seek to live will seek but not play.

Reply:
Also not sure we'd want to wait until the `play` event is fired before doing a seek to live. Per MDN:
> The play event is fired when the paused property is changed from true to false, as a result of the play method, or the autoplay attribute.
The current solution will begin the seek concurrent with the "request to play" (and should be earlier than the media-controller logic, due to bubbling). Your proposed solution would entail that we don't even begin to seek until **_after_** the media element has gone from paused to unpaused.

Reply:
you're right, I think my initial feeling here was that these changes added more complexity than what we've seen in the template.ts file but realizing the reason is Media Chrome. 

The `play` event is indeed later but it would catch any attempt to resume playing. There's cases on iPhone for example with locking and unlocking where the internal video.pause() is called, this wouldn't go through our UI at all and it would not cause the player to jump to the live window. maybe not the worst until we would support DVR.

Reply:
Chiming in here with some comments on the expected behavior (I don't have an opinion on the technical implementation).

First of all, we decided that for `ll-live` and for `live` (non-DVR mode) that tapping the play button will jump to live edge.

**option 1**

- calling `play()` on the mux-player programmatically will also jump to live edge

**option 2**

- calling `play()` on the mux-player programmatically will NOT jump to live edge

Honestly, I don't feel strongly between option 1 and 2 and it feels like a minor detail without a clear expectation of what a developer expects to happen. The good thing is that with option 2 the user will still visually see the "seek to live" button, so nothing would end up in a broken state. For that reason, I'm not too worried about it.

Whichever option is technically simpler is what I think we should go with.